project/apps/collaboration-service/src/adapters/controllers/collaboration.controller.ts

```diff
@@ -29,17 +33,22 @@ export class CollaborationController {
   }
 
   @MessagePattern({ cmd: 'get_collab_info' })
-  async getActiveCollabInfo(@Payload() collabId: string) {
-    return await this.collaborationService.getActiveCollabInfo(collabId);
-  }
-
-  @MessagePattern({ cmd: 'get_collab_history' })
-  async getCollabDocument(@Payload() collabId: string) {
```

Comment:
removed getCollabDocument method, collab document will be retrieved in the new `getAttempts` method.